src/number_theory/padics/padic_numbers.lean

```diff
@@ -88,16 +88,15 @@ let ⟨ε, hε, N1, hN1⟩ := this,
     from lt_max_iff.2 (or.inl this),
   begin
     by_contradiction hne,
-    rw ←padic_norm.neg p (f m) at hne,
-    have hnam := add_eq_max_of_ne p hne,
+    rw ← padic_norm.neg (f m) at hne,
```

Comment:
We also don't have a consensus on `rw ←h` versus `rw ← h`.

Reply:
What about `rw [←h]` or `rw [← h]` for a single rewrite? I find it weird that `simp` and some other things require square brackets but for `rw` it's optional.

Reply:
That's yet another thing we don't have an agreement on.